pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -56,6 +56,14 @@ const (
 	Name = names.DynamicResources
 
 	stateKey framework.StateKey = Name
+
+	// claimParametersGeneratedFromIndex is the lookup name for the index function
+	// which indexes by other resource which generated the claim parameters object.
+	claimParametersGeneratedFromIndex = "claim-parameters-generated-from-index"
+
+	// classParametersGeneratedFromIndex is the lookup name for the index function
+	// which indexes by other resource which generated the class parameters object.
+	classParametersGeneratedFromIndex = "class-parameters-generated-from-index"
```

Comment:
Can we make this shorter and use a single name for both, like "generated-from"?

The name only needs to be unique within one cache, as far as I understand it.